packages/cfd/src/Helpers/compare-accounts-config.ts

```diff
@@ -267,7 +267,6 @@ const getAccountVerficationStatus = (
     switch (jurisdiction_shortcode) {
         case 'synthetic_svg':
         case 'financial_svg':
-        case 'all_svg':
```

Comment:
If you remove this then it will break Swap-free's  add functionality

Reply:
Have you checked the flow of Swap-Free by removing this line?
@shontzu-deriv @hirad-deriv

Reply:
tested, it does not break the flow, however, I can add it back if there is a future concern @hamza-deriv